core/script_language.cpp

```diff
@@ -48,6 +48,24 @@ void Script::_notification(int p_what) {
 	}
 }
 
+bool Script::has_method(const StringName &p_method) const {
+
+	if (Object::has_method(p_method)) {
+		return true;
+	}
+
+	if(defines_method(p_method)) {
```

Comment:
You should have a space between the `if` and the condition.

The next `if` statement also has a lack of a space before its conditional.

Reply:
Fixed.